stagemonitor-core/src/main/java/org/stagemonitor/core/CorePlugin.java

```diff
@@ -573,6 +573,13 @@ private void reportToElasticsearch(Metric2Registry metricRegistry, int reporting
 			reporters.add(reporter);
 			elasticsearchClient.scheduleIndexManagement(ElasticsearchReporter.STAGEMONITOR_METRICS_INDEX_PREFIX,
 					moveToColdNodesAfterDays.getValue(), deleteElasticsearchMetricsAfterDays.getValue());
+		} else if (isOnlyLogElasticsearchMetricReports()) {
```

Comment:
It would be better to not have duplicated code for creating and starting the reporter. Not a must as the amount of duplicated code is quite small but it would be nice.